macros/CSSRef.ejs

```diff
@@ -17,12 +17,12 @@ if (slug) {
     var htmlEscape = kuma.htmlEscape;
     var rtlLocales = ['ar', 'he', 'fa'];
 
-    var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/properties.json";
-    var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/at-rules.json";
-    var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/selectors.json";
-    var typesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/types.json";
-    var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/syntaxes.json";
-    var unitsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/units.json";
+    var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/properties.json";
+    var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/at-rules.json";
+    var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/selectors.json";
+    var typesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/types.json";
+    var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/syntaxes.json";
+    var unitsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/units.json";
```

Comment:
I loaded the anonymized production database, and the sidebar lacks links with that as well, so the macro needs updating.

<img width="350" alt="screenshot 2017-04-05 14 18 17" src="https://cloud.githubusercontent.com/assets/286017/24722804/c18b42ea-1a0a-11e7-8e3d-b933798b9e4e.png">